scripts/setup.sh

```diff
@@ -14,27 +14,35 @@ cd "${SCRIPT_DIR}/.." || echo "Could not cd to repository root"
 # Pinned Ansible version
 ANSIBLE_OK="2.7.8"
 ANSIBLE_VERSION="2.7.11"
+ANSIBLE_REPO_VER="2.7"
+
+as_sudo(){
+   cmd="sudo bash -c '. ${SCRIPT_DIR}/proxy.sh && $1'"
+   ## echo $cmd
+   eval $cmd
+}
```

Comment:
Why use this rather than just sourcing the proxy variables globally in the environment?

Reply:
global variables did not work for all commands executed on local host as well as remote - best option found.